_pages/platform/pipeline.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: article
+title:  "ML Pipelines"
+excerpt: "Creating ML pipelines with Algorithmia."
+categories: basics
+tags: [basics]
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /platform/pipeline/
+---
+
+# ML Pipelining in Algorithmia
+
+Algorithmia was built from the ground up with machine learning at scale use cases in mind. Pipelining is a key part of any full scale deployment solution. Teams need to be able to productionize models as parts of a whole. This means two things: first, splitting up your machine learning workflows into independent, reusable, modular parts that can then be pipelined together to create models. This makes building models more efficient, cutting out redundant work. Second, we want to automate the machine learning workflow by enabling data to be transformed and correlated into a model that can then be analyzed to achieve outputs, streamlining the process of inputting data into the ML model.
```

Comment:
```suggestion
Algorithmia was built from the ground up to support machine learning at scale, and pipelining is a key part of any full-scale deployment solution. Teams must be able to productionize individual models as parts of a larger workflow, which means two things:

1. Complex machine learning workflows should be divided into independent, reusable, modular parts that can then be connected to form a complete pipeline. This sort of modularity makes data scientists and engineers more efficient, cutting out redundant work as the component pieces are reusable and easy to work with.
2. Complex machine learning workflows should be automated. This can be achieved by streamlining the process of transforming data, inputting it into a model where inference happens, and shuttling the output to the next step in the pipeline.
```

Reply:
i moved things around here a bit and wanted to call out the numbered points a bit more by moving them into a list